packages/terra-graphs-docs/src/Css/ExampleGraphContainer.module.scss

```diff
@@ -1,8 +1,12 @@
+.example-graph-container,
```

Comment:
should this file be deleted? I think you moved this to the ExampleGraphContainer folder.